notebooks/official/feature_store/sdk-feature-store.ipynb

```diff
@@ -1281,6 +1281,84 @@
         "After the LRO finishes, you should be able to see the result in the [BigQuery console](https://console.cloud.google.com/bigquery), as a new table under the BigQuery dataset created earlier."
       ]
     },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "## Streaming Ingestion\n",
+      "\n",
+      "Streaming ingestion is currently public preview. \n",
+      "\n",
+      "Streaming ingestion lets you make real-time updates to feature values. While batch import is suitable for importing a large volume of data with high latency, streaming ingestion is suitable for ingesting small amount of data with low latency. The written data will be available to read using batch export and online serving. "
```

Comment:
Rewrite the last sentence as follows (avoid the use of "will"):
"The written data becomes available to read using batch export and online serving."

Reply:
**Similar changes (suggested rewording by line number):**
261 - Remove "will".
504 - "The movie prediction example lets you create..."
526 - "Running the code cell creates..."
579 - "... A process log is printed."
1028 - "By default, the SDK returns..."
1030 - "By default, all the features of an entity type are selected. The output response displays the selected entity type ID..."
1063 - By default, all the features of an entity type are selected.
1170 - "These are used in the next..."

Reply:
Fixed